File: API/Entities/Evaluation.cs
using API.Entities;

public class Evaluation
{
    public int Id { get; set; }
    public int TouristObjectId { get; set; }
    public Guid UserId { get; set; }
    public int TotalPoints { get; set; }
    public int CategoryId { get; set; }
    public DateTime CreatedAt { get; set; }

    public TouristObject TouristObject { get; set; } = null!;
    public User Officer { get; set; } = null!;
    public Category Category { get; set; } = null!;
    public ICollection<EvaluationScore> Scores { get; set; } = new List<EvaluationScore>();
}

File: client/src/store/types/Evaluation.ts
export interface EvaluationScoreDto {
  criteriaId: number;
  criteriaName: string;
  points: number;
}

export interface EvaluationDto {
  id: number;
  totalPoints: number;
  categoryId: number;
  categoryName: string;
  createdAt: string;
  officerFullName: string;
  scores: EvaluationScoreDto[];
}

export interface CreateEvaluationDto {
  touristObjectId: number;
  userId: string; // GUID iz Redux-a
  scores: {
    criteriaId: number;
    points: number;
  }[];
}
